pkgs/tools/security/pynitrokey/default.nix

```diff
@@ -4,47 +4,50 @@
 , nrfutil
 , libnitrokey
 , nix-update-script
+, installShellFiles
 }:
 
 with python3Packages;
```

Comment:
It may be cleaner to use `python3Packages.callPackage` in pkgs/top-level/all-packages.nix:

```diff
-  pynitrokey = callPackage ../tools/security/pynitrokey { };
+  pynitrokey = python3Packages.callPackage ../tools/security/pynitrokey { };
```

and then

```suggestion
# I can't mark it in the suggestion, but also remove python3Packages
, buildPythonApplication
, flit-core
, pythonRelaxDepsHook
, installShellFiles
, certifi
, cffi
, click
, click-aliases
, cryptography
, ecdsa
, fido2
, frozendict
, importlib-metadata
, intelhex
, nethsm
, nkdfu
, protobuf
, pyserial
, python-dateutil
, pyusb
, requests
, semver
, spsdk
, tlv8
, tqdm
, typing-extensions
, libnitrokey
, nix-update-script
}:
 
```